src/css/style.css

```diff
@@ -366,28 +379,28 @@ body {
   color: white;
   font-weight: 500;
   font-size: 1.5rem;
```

Comment:
to display output when you type something, set css property z-index grater then 0 for class .request info